packages/camera/src/components/UploadCenter/motion/index.js

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect, useRef } from 'react';
+import { Animated, StyleSheet } from 'react-native';
+import PropTypes from 'prop-types';
+
+const Motion = () => {};
+
+function Initiator({ children, style, extraData, maxOpacity, minOpacity, ...props }) {
+  const opacityMotion = useRef(new Animated.Value(0)).current;
+
+  useEffect(() => {
+    Animated.timing(opacityMotion, { toValue: minOpacity, useNativeDriver: true, duration: 50 })
+      .start(
+        () => Animated.timing(opacityMotion, { toValue: maxOpacity, useNativeDriver: true })
+          .start(),
+      );
+  }, [...extraData]);
+
```

Comment:
And basically this component will animate whenever the extraData array got changed.